shared/reducers/fs.js

```diff
@@ -9,7 +10,9 @@ export default function(state: Types.State = initialState, action: FSGen.Actions
   switch (action.type) {
     case FSGen.resetStore:
       return initialState
-    case FSGen.increaseCount:
+    case FSGen.folderListLoaded:
+      return state.setIn(['pathItems'], I.merge(state.pathItems, action.payload.pathItems))
```

Comment:
can this be expressed as a mergeIn

Reply:
I was hoping there was a `mergeIn` but didn't find it in immutable.js . Google FAIL!